deeplabcut/refine_training_dataset/stitch.py

```diff
@@ -606,6 +606,9 @@ def compute_max_gap(tracklets):
         return max_gap
 
     def mine(self, n_samples):
+        if not self._lu_overlap:
+            raise ValueError("No overlapping tracklets found.")
```

Comment:
Maybe add more information suggesting it's because only one animal is found from the assembly?

Reply:
Good point! Then I don't have other questions